plugins/org.eclipse.thym.android.core/META-INF/MANIFEST.MF

```diff
@@ -2,13 +2,13 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %BundleName
 Bundle-SymbolicName: org.eclipse.thym.android.core;singleton:=true
-Bundle-Version: 2.0.0.qualifier
+Bundle-Version: 2.1.0.qualifier
 Bundle-Activator: org.eclipse.thym.android.core.AndroidCore
 Bundle-Vendor: %BundleVendor
 Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.core.resources;bundle-version="[3.8.1,4.0.0)",
  org.eclipse.debug.core;bundle-version="[3.7.0,4.0.0)",
- org.apache.commons.io;bundle-version="[2.0.0,3.0.0)",
+ org.apache.commons.io;bundle-version="[2.1.0,3.0.0)",
```

Comment:
Is this intentional or a search/replace mistake? There are similar suspicious changes regarding various version numbers, can you confirm that they are intentional